### DISCUSSION TITLE
I would like to suggest to have `preprocess.rs` as part of the library in the next version release

### DISCUSSION THREAD

**Discussion by Raj2032** [robertknight/ocrs#71] [Ideas]
Problem is is that I have to copy the entire `preprocess.rs` file just so I can then do this:

```rust
let img_source = ocrs_extras::preprocess::ImageSource::from_bytes(img.as_raw(), img.dimensions()).unwrap();`
```

in the `main()` function. It would make more sense to have `preprocess.rs` in the library of `ocrs`.

**Comment by robertknight** [comment]
If you are using `ocrs` from the `main` branch, you can import `ImageSource` like this:

```
use ocrs::ImageSource
```

See [the hello_ocr example](https://github.com/robertknight/ocrs/blob/main/ocrs/examples/hello_ocr.rs) from the main branch.

If you are using the most recently published version of ocrs on `crates.io`, `ImageSource` doesn't exist and instead you would load the image using the code shown in [the corresponding version of the hello_ocr example]( https://github.com/robertknight/ocrs/blob/ocrs-v0.6.0/ocrs/examples/hello_ocr.rs).

Sticking with the published version of the crate and the corresponding code from the older version of the example is the best way to go at the moment, until new changes on the `main` branch are finalized.

**Reply by Raj2032** [reply]
right I see mate thanks
